examples/cnab-simple/README.md

```diff
@@ -1,60 +1,127 @@
-# Docker Application to CNAB
+# Example: From Docker App to CNAB
```

Comment:
I wonder if all this example should be merged with the hello world example, as it does not show more things than the hello-world example already has, in its `.dockerapp` structure 🤔

Reply:
Yes I agree. I would not remove it. But we might want to use another simple App definition. Any proposal?

Reply:
We could maybe merge it as it is, then think about proposing another app definition in another step?

Reply:
Ok for a follow-up 👍